rdr_service/genomic/genomic_job_components.py

```diff
@@ -2619,15 +2539,19 @@ def _get_source_data_query(self, manifest_type):
                     ]
                 ).select_from(
                     sqlalchemy.join(
-                        sqlalchemy.join(ParticipantSummary,
-                                        GenomicSetMember,
-                                        GenomicSetMember.participantId == ParticipantSummary.participantId),
-                        GenomicGCValidationMetrics,
-                        GenomicGCValidationMetrics.genomicSetMemberId == GenomicSetMember.id
+                        sqlalchemy.join(
+                            sqlalchemy.join(ParticipantSummary,
+                                            GenomicSetMember,
+                                            GenomicSetMember.participantId == ParticipantSummary.participantId),
+                            GenomicGCValidationMetrics,
+                            GenomicGCValidationMetrics.genomicSetMemberId == GenomicSetMember.id
+                        ),
+                        GenomicFileProcessed,
+                        GenomicFileProcessed.id == GenomicSetMember.aw1FileProcessedId
```

Comment:
This is purely a styling suggestion. This:
```
sqlalchemy.join(
    sqlalchemy.join(
        sqlalchemy.join(ParticipantSummary,
                        GenomicSetMember,
                        GenomicSetMember.participantId == ParticipantSummary.participantId),
        GenomicGCValidationMetrics,
        GenomicGCValidationMetrics.genomicSetMemberId == GenomicSetMember.id
    ),
    GenomicFileProcessed,
    GenomicFileProcessed.id == GenomicSetMember.aw1FileProcessedId
)
```

could be re-written as
```
sqlalchemy.join(
    ParticipantSummary,
    GenomicSetMember,
    GenomicSetMember.participantId == ParticipantSummary.participantId
).join(
    GenomicGCValidationMetrics,
    GenomicGCValidationMetrics.genomicSetMemberId == GenomicSetMember.id
).join(
    GenomicFileProcessed,
    GenomicFileProcessed.id == GenomicSetMember.aw1FileProcessedId
)
```

If you've already thought of it and like the nesting better for any reason then definitely ignore me. But it feels like it could potentially make it a little easier to visualize the query structure.

Reply:
Thanks, I like that suggestion. I will update.